lib/recurly/custom_field.rb

```diff
@@ -0,0 +1,17 @@
+module Recurly
+  # Delivery objects are only used with GiftCards.
+  #
+  # Recurly Documentation: https://dev.recurly.com/docs/custom-fields
```

Comment:
meant to update this, not sure where it will live

Reply:
This should be fine for now. I'll update it if it's different before release.